app/controllers/kondos_controller.rb

```diff
@@ -13,4 +13,45 @@ def index
   def show
     @kondo = Kondo.find(params[:id])
   end
+
+  def new
+    @kondo = Kondo.new
+  end
+
+  def edit
+  end
+
+  def create
+    @kondo = Kondo.new(kondo_params)
+
+    if @kondo.save
+      redirect_to kondo_path(@kondo), notice: 'Kondo was successfully created.'
+    else
+      render :new
+    end
+  end
+
+  def update
+    @kondo.update(kondo_params)
+    if @kondo.save
+      redirect_to path(@kondo), notice: 'Kondo was successfully updated.'
```

Comment:
Last thing, would `path(@kondo)` redirect to the show page of the kondo too? If not then I think it should be the same with save where its `redirect_do kondo_path(@kondo)`.